modules/ensemble/pubspec.yaml

```diff
@@ -141,6 +141,7 @@ dependencies:
   flutter_slidable: ^3.1.1
   accordion: ^2.6.0
   session_storage: ^0.0.1
+  moengage_flutter: ^8.0.0
```

Comment:
make sure when you make a build for both android and IOS to test it does not need any app entitlement

Reply:
- For iOS, have texted you about testing.

- For android, I am not sure, wot entitlement is mentioned here, but it you are talking about configurations, while making build we have to update the following files for android for which I have created script:
1. `ensemble.config`: for adding moenagageAppID
2. app level gradle file to include moenageSDK implementation
3. Manifest file
4. `MainActivity.kt` and also add 2 new `.kt` files for handling notifications callback functions.